agnpy/absorption/absorption.py

```diff
@@ -443,6 +445,96 @@ def evaluate_tau_blr_mu_s(
         )
         return (prefactor * integral).to_value("")
 
+    @staticmethod
+    def evaluate_tau_blr_cubepy(
+        nu,
+        eps_abs,
+        z,
+        mu_s,
+        L_disk,
+        xi_line,
+        epsilon_line,
+        R_line,
+        r,
+        mu=mu_to_integrate,
```

Comment:
the `mu` and `phi` arguments are not used in the function, you use directly the default `mu_to_integrate` and `phi_to_integrate`

Reply:
Done